izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerFrame.java

```diff
@@ -274,6 +274,8 @@ public void buildGUI()
         this.setDefaultCloseOperation(WindowConstants.DO_NOTHING_ON_CLOSE);
         ImageIcon jframeIcon = getIcons().get("JFrameIcon");
         setIconImage(jframeIcon.getImage());
+        setWMClass(getTitle());
```

Comment:
You could add a check if running under Linux before call `setWMClass`:
`if (!installData.getPlatform().isA(Platform.Name.LINUX))`

Reply:
Thanks, done.